src/material-experimental/mdc-slider/slider.ts

```diff
@@ -335,7 +338,7 @@ export class MatSlider implements AfterViewInit, OnChanges, OnDestroy, ControlVa
         // client rectangle wouldn't reflect the new directionality.
         // TODO(devversion): ideally the MDC slider would just compute dimensions similarly
         // to the standard Material slider on "mouseenter".
-        this._ngZone.runOutsideAngular(() => setTimeout(() => this._foundation.layout()));
+        setTimeout(() => this._foundation.layout());
```

Comment:
@mmalerba To move our discussion on Slack to here. Running this outside of the Angular zone means that the task is not captured by the Angular zone and testing zone. 

This means that we cannot flush the task (within `fakeAsync`), nor can we wait on the zone becoming stable. We should definitely not degrade performance in favor of test harnesses, so we need to find a solution for handling such scenarios.